framework/wazuh/core/requests.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GP
+import os
+import ssl
+from datetime import datetime
+from logging import getLogger
+from typing import Optional
+
+import aiohttp
+import certifi
+
+import wazuh
+from wazuh.core.configuration import get_ossec_conf
+from wazuh.core.utils import get_utc_now
+
+CTI_URL = get_ossec_conf(section='global').get(
+    'cti_url', 'http://cti:4041'
+)  # This default must be removed once we have the configuration in the ossec parser.
+RELEASE_UPDATES_URL = os.path.join(CTI_URL, 'api', 'v1', 'ping')
+ONE_DAY_SLEEP = 60 * 60 * 24
+WAZUH_UID_KEY = 'wazuh-uid'
+WAZUH_TAG_KEY = 'wazuh-tag'
+
+
+logger = getLogger('wazuh')
+
+
+def _get_connector() -> aiohttp.TCPConnector:
+    """Return a TCPConnector with default ssl context.
+
+    Returns
+    -------
+    aiohttp.TCPConnector
+        Instance with default ssl connector.
+    """
+    ssl_context = ssl.create_default_context(cafile=certifi.where())
+    return aiohttp.TCPConnector(ssl=ssl_context)
+
+
+def get_update_information_template(
+        update_check: bool,
+        current_version: str = wazuh.__version__,
+        last_check_date: Optional[datetime] = None
+) -> dict:
+    """Build and return a template for the update_information dict.
+
+    Parameters
+    ----------
+    update_check : bool
+        Indicates if the check is enabled or not.
+    current_version : str, optional
```

Comment:
Since the `current_version` has a default value that will always be set in `framework/wazuh/__init__.py`, the optional type might not be needed. Could a `None` value be passed to it?

Reply:
Here too, this is generated according to the numpy docstring format.